src/azul/service/manifest_controller.py

```diff
@@ -137,28 +140,41 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            unix_cmd = f'curl "{location}"'
+            windows_cmd = f'curl.exe "{location}"'
             body = {
                 'Status': 301,
                 'Location': location.url,
                 'Retry-After': token.wait_time
             }
         else:
-            if fetch:
+            try:
+                query_params['objectKey']
+            except KeyError:
                 url = self.manifest_url_func(fetch=False,
                                              catalog=manifest.catalog,
                                              format_=manifest.format_,
                                              filters=json.dumps(manifest.filters),
                                              objectKey=manifest.object_key)
             else:
                 url = manifest.location
+            command_lines = self.service.command_lines(manifest, url)
+            unix_cmd = command_lines['bash']
+            windows_cmd = command_lines['cmd.exe']
             body = {
                 'Status': 302,
                 'Location': url,
-                'CommandLine': self.service.command_lines(manifest, url)
+                'CommandLine': command_lines
             }
-
+        instructions = self._curl_instructions(unix_cmd, windows_cmd)
         if fetch:
             return Response(body=body)
         else:
             headers = {k: str(body[k]) for k in body.keys() & {'Location', 'Retry-After'}}
-            return Response(body='', status_code=body['Status'], headers=headers)
+            return Response(body=instructions, status_code=body['Status'], headers=headers)
+
+    def _curl_instructions(self, unix_cmd, windows_cmd) -> str:
+        return ''.join([dedent(f'''
+            Download the manifest with `curl` in {os} OS using:
+            {cmd}
+            ''') for os, cmd in (('Unix', unix_cmd), ('Windows', windows_cmd))])
```

Comment:
Note that `[]` aren't needed when a comprehension is the only argument to a function call. 

```suggestion
    def _curl_instructions(self, cmds_by_os: Mapping[str, str]) -> str:
        return ''.join(
            dedent(f'''
                Download the manifest with `curl` in {os} OS using:
                {cmd}
            ''') 
            for os, cmd in cmds_by_os.items()
        )
```
Consider using `sorted(cmds_by_os.items())` if the order of the commands is important, e.g. for verifying the response in tests.

This still has trailing and leading newlines but you thumbs-upped that comment so I'm guessing that's intentional.